tasks/deps-rh-74.yml

```diff
@@ -0,0 +1,66 @@
+---
+
+- name: "Install epel repo"
+  yum:
+    name: "epel-release"
+    state: "installed"
+
+- name: "Enable epel repo"
+  ini_file:
+    path: /etc/yum.repos.d/epel.repo
+    section: epel
+    option: enabled
+    value: 1
+
+- name: "Import Remi GPG key (AtoM version >= 7.4)"
```

Comment:
(another nitpick) Should the name of the task be something like ".... (php version >= 7.4)" instead of "....  (AtoM version >= 7.4)" ?